editor/gui/editor_quick_open_dialog.cpp

```diff
@@ -45,6 +45,14 @@
 #include "scene/gui/texture_rect.h"
 #include "scene/gui/tree.h"
 
+const Vector2i MAGIC_HIGHLIGHT_OFFSET = { 4, 5 };
+const String DISPLAY_MODE_SETTING = "filesystem/quick_open_dialog/default_display_mode";
+const String FUZZY_MATCHING_SETTING = "filesystem/quick_open_dialog/enable_fuzzy_matching";
+const String INCLUDE_ADDONS_SETTING = "filesystem/quick_open_dialog/include_addons";
+const String MAX_RESULTS_SETTING = "filesystem/quick_open_dialog/max_results";
+const String MAX_MISSES_SETTING = "filesystem/quick_open_dialog/max_fuzzy_misses";
+const String SEARCH_HIGHLIGHT_SETTING = "filesystem/quick_open_dialog/show_search_highlight";
```

Comment:
We don't really use setting constants like this. Especially when they are used only once.